chapters/4-acquisition.tex

```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
 
 
 At this point, it is useful to do a \textbf{survey pilot}\sidenote{
- \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}\index{survey pilot}
+  More details on how to structure survey pilots,
+	the different components of comprehensive survey pilot and
+	link to many other resources can be found on the DIME Wiki:
+ \url{https://dimewiki.worldbank.org/Survey\_Pilot}}\index{survey pilot}
 of the questionnaire.
-A comprehensive survey pilot is a critical stage in survey design.\sidenote{
-	\url{https://dimewiki.worldbank.org/Survey\_Pilot}}
+A comprehensive survey pilot is a critical stage in survey design.
 The pilot must be done out-of-sample,
 but in a context as similar as possible to the study sample.
 The survey pilot includes three steps:
-a \textbf{pre-pilot}, a \textbf{content-focused pilot}, and a \textbf{data-focused pilot}.\sidenote{
-	\url{https://dimewiki.worldbank.org/Structuring\_a\_Survey\_Pilot}}
+a \textbf{pre-pilot}, a \textbf{content-focused pilot}, and a \textbf{data-focused pilot}.
 The first step is a \textbf{pre-pilot}.
 The pre-pilot is a qualitative exercise, done early in the questionnaire design process.
 The objective is to answer broad questions about how to measure key outcome variables,
 and gather qualitative information relevant to any of the planned survey modules.
 A pre-pilot is particularly important when designing new survey instruments.
 
-The second step is a \textbf{content-focused pilot}\sidenote{
-  \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}.
+The second step is a \textbf{content-focused pilot}.
 The objectives at this stage are to improve the structure and length of the questionnaire,
 refine the phrasing and translation of specific questions,
 and confirm coded response options are exhaustive.\sidenote{
+  A checklist for content-focused pilots can be found on the DIME Wiki:
 	\url{https://dimewiki.worldbank.org/index.php?title=Checklist:\_Refine\_the\_Questionnaire\_(Content)}}
-In addition, it is an opportunity to test and refine all survey protocols,\sidenote{
-	\url{https://dimewiki.worldbank.org/Piloting\_Survey_Protocols}}
+In addition, it is an opportunity to test and refine all survey protocols,
```

Comment:
Add here: "For a detailed discussion of testing survey protocols, go to https://dimewiki.worldbank.org/wiki/Piloting_Survey_Protocols" BUT @Avnish95 is changing the name so once that is done will be Checklist: Piloting Survey Protocols

Reply:
done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a